docs/api-types/certificate/README.md

```diff
@@ -0,0 +1,32 @@
+# Certificates
+
+cert-manager has the concept of 'Certificates' that define a desired X.509 certificate. `Certificate` resources reference an `Issuer` or `ClusterIssuer` for information on how to obtain the certificate.
+
+A simple `Certificate` could be defined as:
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: acme-crt
+spec:
+  secretName: acme-crt
+  commonName: foo.example.com
+  dnsNames:
+  - bar.example.com  
+  issuerRef:
+    name: letsencrypt-prod
+    # We can reference ClusterIssuers by changing the kind here.
+    # The default value is Issuer (i.e. a locally namespaced Issuer)
+    kind: Issuer
+```
+
+This `Certificate` will tell cert-manager to attempt to use the referenced `letsencrypt-prod` `Issuer` resource to obtain a certificate key pair for the `foo.example.com` domain. If successful, the resulting key and certificate will be stored in a secret named `acme-crt` with keys of `tls.key` and `tls.crt` respectively. This secret will live in the same namespace as the `Certificate` resource.
+
+The `dnsNames` field specifies a list of [Subject Alternative Names][1] to be associated with the certificate. If the `commonName` field is omitted, the first element in the list will be the common name.
+
+## Namespacing
+
+A `Certificate` is a namespaced resource. This means that the referenced `Issuer` must exist in the same namespace as the `Certificate`. If you want to create a `Certificate` that can be used in any namespace, you should consider creating and referencing a `ClusterIssuer` resource.
```

Comment:
Needs updating. It's worth noting that a Certificate is namespaced, but there's not much more to add aside from maybe that a Certificate can be issued by a ClusterIssuer (although that might already be clear enough from the first paragraph)